sa/db/boulder_sa/20220328100000_Incidents.sql

```diff
@@ -10,29 +10,7 @@ CREATE TABLE `incidents` (
     PRIMARY KEY (`id`)
 ) CHARSET=utf8mb4;
 
-CREATE TABLE `incident_foo` (
```

Comment:
Modifying an existing migration (as opposed to adding another one) feels a little weird, but in this case I think it makes sense -- if someone already has these tables in place they'll continue to exist and that's okay, and if someone deletes their test db then they'll never be recreated and that's also good.

Can you make sure that the commit description includes a note that the corresponding migration to create these tables in the incidents_sa schema was already added in the previous PR? I had to think to remember why this change is lopsided.

Reply:
Good point. I've edited the PR description to include that these tables were added in #6344.